src/core/ext/transport/inproc/inproc_transport.c

```diff
@@ -72,6 +72,7 @@ typedef struct sb_list_entry {
 typedef struct {
   grpc_byte_stream base;
   sb_list_entry *le;
+  grpc_error *shutdown_error;
```

Comment:
Maybe it makes sense to pull this into the base struct? I know `grpc_chttp2_incoming_byte_stream` doesn't have it at the moment, but maybe it should based on my other comments